dev/devicelab/lib/framework/utils.dart

```diff
@@ -498,3 +500,36 @@ int parseServicePort(String line) {
   final Match match = _kObservatoryRegExp.firstMatch(line);
   return match == null ? null : int.parse(match.group(2));
 }
+
+/// If FLUTTER_ENGINE environment variable is set then we need to pass
+/// correct --local-engine setting too.
+void setLocalEngineOptionIfNecessary(List<String> options, [String flavor]) {
```

Comment:
Who was meant to call this method with `flavor` parameter specified? None of the five call sites below specify that `flavor` option.
I'm curious because I normally use `android_debug_unopt`, rather than `android_debug`.

Reply:
There was a call site passing 'debug', but I removed it. Should have maybe killed the parameter `flavor` too, but decided to keep it for potential future use. 

Is `android_debug_unopt` a good variant when you do performance measurements though?

Reply:
> Is android_debug_unopt a good variant when you do performance measurements though?

As long as you consistently use that variant, I think it is good indication of hot reload performance, yes. We also refer to that variant on flutter docs(https://github.com/flutter/flutter/blob/master/CONTRIBUTING.md, for example).